src/components/Chart/LineChart.jsx

```diff
@@ -306,15 +306,33 @@ class LineChart extends Component {
   }
 
   render() {
-    const { width, height, gradient, margin } = this.props
+    const { width, height, gradient, margin, container } = this.props
     const { selectedItem, x } = this.state
 
+    const containerWidth = get(container, 'current.offsetWidth')
```

Comment:
Do not use `offsetWidth` inside `render` since this will trigger a reflow : https://gist.github.com/paulirish/5d52fb081b3570c81e3a#box-metrics.